app/views/content/how-to-apply-for-teacher-training/subject-knowledge-enhancement.md

```diff
@@ -49,11 +54,11 @@ Find out more about [what it’s like to teach in secondary](/life-as-a-teacher/
 
 ### The benefits of an SKE course
 
-Learn more about how a subject knowledge enhancement course can help prepare you for the classroom in our video:
+An SKE allows you to build up your subject knowledge that you'd be expected to have as a teacher. For example, if you want to teach a subject that you're passionate about, but you only have an A Level in the subject and your degree isn't directly related. Your training provider might suggest an SKE to help you improve your subject knowledge and feel more confident starting your teacher training.
 
-$ske-course-video$
+$quote-confidence$
 
- [Read the transcript for the 'Subject knowledge enhancement explained' video](/how-to-apply-for-teacher-training/ske-explained-transcript).
+$quote-maths$
```

Comment:
Is it worth splitting up the quotes so they are in different sections? Feels slightly odd to have one on top of the other - could one go (for example) at the end of the SKE subjects section?